textdb/textdb-sandbox/src/main/java/edu/uci/ics/textdb/sandbox/team3lucenequeryexample/LuceneQueryExample.java

```diff
@@ -0,0 +1,78 @@
+package edu.uci.ics.textdb.sandbox.team3lucenequeryexample;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+import org.apache.lucene.analysis.Analyzer;
+import org.apache.lucene.analysis.Tokenizer;
+import org.apache.lucene.analysis.ngram.NGramTokenizer;
+import org.apache.lucene.document.Document;
+import org.apache.lucene.document.Field;
+import org.apache.lucene.document.FieldType;
+import org.apache.lucene.index.DirectoryReader;
+import org.apache.lucene.index.IndexOptions;
+import org.apache.lucene.index.IndexWriter;
+import org.apache.lucene.index.IndexWriterConfig;
+import org.apache.lucene.queryparser.classic.QueryParser;
+import org.apache.lucene.search.IndexSearcher;
+import org.apache.lucene.search.Query;
+import org.apache.lucene.search.ScoreDoc;
+import org.apache.lucene.search.TopDocs;
+import org.apache.lucene.store.Directory;
+import org.apache.lucene.store.FSDirectory;
+
+public class LuceneQueryExample {
+	public static void main(String[] args) throws Exception {
+		String dataFileName = "/Users/laishuying/Desktop/lucenetext.txt";
```

Comment:
We shouldn't use hard-coded paths.  I wonder whether your test case really works with this path that doesn't exist on other machines.

It's better to use a relative path and put the txt file to git as well.  Make sure it's not big.